image-c9s.yaml

```diff
@@ -24,3 +24,6 @@ vmware-hw-version: 15
 aws-imdsv2-only: false
 aws-volume-type: "gp2"
 aws-x86-boot-mode: "legacy-bios"
+
+# add the requisite OCP metadata to our container image
+ostree-container-inject-openshift-cvo-labels: true
```

Comment:
see https://github.com/openshift/os/blob/master/image-c9s.yaml#L9
at this point we could probably drop one of the two files.

Reply:
Ahh whoops, didn't realize it was already there. I just dropped that hunk for now rather than try to include a larger change in this PR.